gem_city.rb

```diff
@@ -0,0 +1,38 @@
+# This class represents the town of GemCity
+# This is a town riddled with crime but we can find out how happy the town is
+class GemCity
+  attr_reader :population
+  attr_accessor :officers, :thieves
+
+  def initialize
+    @officers = 1
+    @thieves = 5
+    @population = 50
+  end
+
+  def thieves(thieves_number = @thieves)
+    @thieves = thieves_number
+  end
```

Comment:
I did not change this to the default setter. I'm not supposed to do so right? I will need to change the call in the spec as well.